### ISSUE TITLE
Automatic initial network creation improvement

### ISSUE THREAD

**Issue by Dillton** [Stephan-S/FS22_AutoDrive#53]
Hello AD team.
First I would like to thank you for great work on AD for FS22.
Second I am not sure if this should be posted in issues section but I have a little improvement in my mind for initial network creation which I think is already very useful but I think there is a core function missing now which I think would be very useful.

Simply would be nice to have inside the confirmation window for initial network creation an option to **adjust course offset** for two way roads.
What is that mean? Many maps have narrower two way roads which results in collisions or traffic jams (mostly caused by wide machinery) and fix for this situations now is to move manually the courses more to the side of the road so for example half of the machine is off the road at the end.
If I am not mistaken and from what I have seen ingame the courses are now generated in the middle of each way. If not or if its already somehow considered during the initial creation please let me know.

So basically this is the whole idea just to have an option to adjust course offset to right or left.
Once again thank you very much for great work and Merry Christmas. ;)

**Comment by Iwan1803** [comment]
The problem with an offset: in a curve the vehicles come too far outside or inside = wall, fences, trees, lamps, NPC...

**Comment by Dillton** [comment]
I agree with you however in this cases it will not be that much time consuming to adjust a few places on map where the collisions occur because of trees lamps etc. in compare with collisions almost everywhere on narrow roads.
Also when there will be some tree or lamp etc. great solution for this is the map object hider mod which I was using in FS19 and I think its must have mod in combination with AD.

Actually everyone would be able to choose what they preffer. Who dont want to use offset just leaves it default = 0 for example. Who finds it helpful can use it. If you know where I am heading. It would be upon everyones decision :)

But as I said before its just an idea however I am still conviced most of us who using AD would finds it very helpful ;)

**Comment by Axel32019** [comment]
No further improvement planned -> closed
